File: exercises/beecrowd/1182/1182.ts
// https://judge.beecrowd.com/pt/problems/view/1182
import * as fs from "fs";

function main(): void {
  const input = fs.readFileSync("/dev/stdin", "utf8").trim().split("\n");
  const c = parseInt(input[0]);
  const op = input[1];

  const values = input.slice(2).map(Number);
  let sum = 0;

  for (let i = 0; i < 12; i++) {
    sum += values[i * 12 + c];
  }

  if (op === "M") {
    sum /= 12;
  }

  console.log(sum.toFixed(1));
}

main();

File: exercises/beecrowd/1182/1182.go
// https://judge.beecrowd.com/pt/problems/view/1182
package main

import "fmt"

func main() {
	var c int
	var op string
	fmt.Scan(&c)
	fmt.Scan(&op)

	matrix := make([][]float64, 12)
	for i := 0; i < 12; i++ {
		matrix[i] = make([]float64, 12)
		for j := 0; j < 12; j++ {
			fmt.Scan(&matrix[i][j])
		}
	}

	sum := 0.0
	for i := 0; i < 12; i++ {
		sum += matrix[i][c]
	}

	if op == "M" {
		sum /= 12.0
	}

	fmt.Printf("%.1f\n", sum)
}
